test/ModuleInterface/fixed-layout-property-initializers.swift

```diff
@@ -0,0 +1,67 @@
+// RUN: %empty-directory(%t)
+
+// RUN: %target-swift-frontend -emit-interface-path %t.swiftinterface -emit-module -o /dev/null %s
+// RUN: %FileCheck %s < %t.swiftinterface --check-prefix CHECK --check-prefix COMMON
+
+// RUN: %target-swift-frontend -emit-interface-path %t-resilient.swiftinterface -enable-resilience -emit-module -o /dev/null %s
+// RUN: %FileCheck %s --check-prefix RESILIENT --check-prefix COMMON < %t-resilient.swiftinterface
+
+// FIXME(rdar44993525): %target-swift-frontend -emit-module -o %t/Test.swiftmodule %t.swiftinterface -disable-objc-attr-requires-foundation-module
+// FIXME(rdar44993525): %target-swift-frontend -emit-module -o /dev/null -merge-modules %t/Test.swiftmodule -module-name Test -emit-interface-path - | %FileCheck %s --check-prefix CHECK --check-prefix COMMON
```

Comment:
For my own curiosity, what goes wrong here?

Reply:
Since we don't print initializers in a non-resilient module, compiling the emitted interface still fails with:

```
error: return from initializer without initializing all stored properties
```

Reply:
The presence of the @hasInitialValue attribute should be the same as an initializer in a non-resilient module, since the initializer expression has a public entry point in the dylib